### ISSUE TITLE
Not, always, including merged in commits

### ISSUE THREAD

**Issue by tomasbjerre** [tomasbjerre/git-changelog-lib#29]
In this scenario:
```
*       a3f3de2 2016-03-08  Polish (Phillip Webb)
*       644ae2c 2016-03-08  Merge branch '1.3.x' (Phillip Webb)
|\  
| *     5e722da 2016-03-08 (origin/1.3.x)  Polish (Phillip Webb)
* |     d7d2404 2016-03-08  Ensure that Tomcat does not report unstopped main thread when startup fails (Andy Wilkinson)
* |     cc2f6f4 2016-03-08  Merge branch '1.3.x' (Stephane Nicoll)
```

When creating log from *d7d2404* to *644ae2c*. It will not include *5e722da*!

**Comment by coderzh** [comment]
Yes, this is the exactly problem.

**Comment by tomasbjerre** [comment]
Made a release of the lib and the jenkins plugin now. Should be fixed in jenkins plugin 1.29. Available from the update sites within a few hours!

**Comment by coderzh** [comment]
Many thanks! It works perfectly!
